package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@lifesg/react-design-system",
-    "version": "1.0.0-alpha.11",
+    "version": "1.0.0-alpha.12",
```

Comment:
just want to double check if there are any more incoming changes? if yes we can bundle all the changes together and bump it in one go